### PR TITLE
Allow bookinfo services domain to be configurable

### PR DESCRIPTION
**Pull request by zlav**
This is needed to run on CloudFoundry

Signed-off-by: Zachary Gershman <zgershman@pivotal.io>

### CODE REVIEW

File: samples/bookinfo/src/productpage/productpage.py [hunk 1]
```diff
@@ -44,26 +45,29 @@
 from flask_bootstrap import Bootstrap
 Bootstrap(app)
 
+servicesDomain = "" if (os.environ.get("SERVICES_DOMAIN") == None) else "." + os.environ.get("SERVICES_DOMAIN")
+servicesPort = "9080"
```
Comment (frankbu) [inline samples/bookinfo/src/productpage/productpage.py]: why is servicesPort moved to a variable if it's not configurable anyway?

File: samples/bookinfo/src/productpage/productpage.py [hunk 2]
```diff
@@ -44,26 +45,29 @@
...
+
 details = {
-    "name" : "http://details:9080",
+    "name" : "http://{0}{1}:{2}".format("details", servicesDomain, servicesPort),
```
Comment (frankbu) [inline samples/bookinfo/src/productpage/productpage.py]: nit, more clear like this: "http://details{0}:{1}".format(servicesDomain, servicesPort)

**Comment by zlav** [comment]
Hi @frankbu The googlebot removed the cla:yes tag after I added another committer but I removed them so it should be all good. I was initially verified in the beginning of the request and if you need to reverify that I signed it I can send you the verification

**Comment by aaronshurley** [comment]
@frankbu please let us know if there's anything else to do. I don't think our changes would've affected codecov or racetest

**Comment by rshriram** [comment]
as part of this change, can you also change all the bookinfo.yamls to point to the new version of all bookinfo images? changes to bookinfo code are not tested, until they are docker-fied

**Comment by zachgersh** [comment]
Closed in favor of #5483

**Pull request closed without merging**